collect_app/src/main/java/org/odk/collect/android/utilities/ToastUtils.java

```diff
@@ -1,37 +1,61 @@
 package org.odk.collect.android.utilities;
 
+import android.view.Gravity;
 import android.widget.Toast;
 
 import org.odk.collect.android.application.Collect;
 
-
 public class ToastUtils {
 
     private ToastUtils() {
 
     }
 
     public static void showShortToast(String message) {
-        showToast(message, Toast.LENGTH_SHORT);
+        showToast(message, Toast.LENGTH_SHORT, false);
     }
 
     public static void showShortToast(int messageResource) {
-        showToast(messageResource, Toast.LENGTH_SHORT);
+        showToast(messageResource, Toast.LENGTH_SHORT, false);
     }
 
     public static void showLongToast(String message) {
-        showToast(message, Toast.LENGTH_LONG);
+        showToast(message, Toast.LENGTH_LONG, false);
     }
 
     public static void showLongToast(int messageResource) {
-        showToast(messageResource, Toast.LENGTH_LONG);
+        showToast(messageResource, Toast.LENGTH_LONG, false);
+    }
+
+    public static void showShortToastCentered(String message) {
```

Comment:
What's special about these isn't so much that they're horizontally centered which is what "centered" implies but that they're vertically centered. How about calling these `showxxxInMiddle`.